grind/bin/grind

```diff
@@ -355,6 +355,10 @@ class TestRunner:
                             type=int,
                             default=default_timeout,
                             help="Per-task timeout. The default timeout is %s seconds." % default_timeout)
+        parser.add_argument('--java-version',
+                            type=int,
+                            default=7,
```

Comment:
Existing grind users are compiling CDH5 Hadoop, which supports JDK8 as a runtime version. I think it's safe to default to 8. The upside is that grind will work better out-of-the-box with today's Java programs.